WordPress/src/main/java/org/wordpress/android/models/Blog.java

```diff
@@ -115,6 +122,15 @@ public URI getUri() {
         }
     }
 
+    public String getHomeURLHost() {
```

Comment:
This routine seems like a good candidate for either `UrlUtils.java` or `WPUrlUtils.java`.